getting-started/routing.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Routing"
+weight: 30
+---
+
+# Routing
+
+Routing is an algorithm which defines how chats on the license are distributed between the agents. This document describes chat.io product routing but in the future you will be able to write your own routing algorithm and run it on our platform.
+
+
+### Introduction
+
+In chat.io product it is not possible for a customer to have multiple chats within a single license, the conversation with a license (company) is continous from customer perspective and it is not nessesary to split that into multiple chats. Agents are assigned to this chat automaticly, or manually depends on routing setting.
+
+
+### Automatic routing
+
+
+![Automatic routing](../images/automatic-routing.png "automatic routing")
+
+
+### Manual routing
+
+
+![Manual routing](../images/manual-routing.png "manual routing")
+
+
+### System messages
+
+While routing switch states (as on diagrams) router will send some system messages (see [Events](../events#system-message) section) to chat.
+
+| Message text | System message type |
+|--------------|---------------------|
+| `Chat assigned to <agent_name>` | `routing.assigned` |
+
+cases:
+
+ - new chat and agent available (automatic routing)
```

Comment:
new chat and agent available (automatic routing) -> new chat and an agent is available (automatic routing)